hook/lua/system/Blueprints.lua

```diff
@@ -81,6 +81,6 @@ do  -- Hook Start
                 end
             end
         end
-        table.remove(all_bps.Projectile['/mods/4dc/projectiles/sniper_piercing/sniper_piercing_proj.bp'].Categories, 2)
+        --table.remove(all_bps.Projectile['/mods/4dc/projectiles/sniper_piercing/sniper_piercing_proj.bp'].Categories, 2)
```

Comment:
Once I merge commit the BP.lua, This will probably conflict so We'll just have to make sure this is deconflicted during the merge process 👍